ghost/core/test/e2e-api/admin/__snapshots__/pages.test.js.snap

```diff
@@ -1698,7 +1698,7 @@ We've crammed the most important information to help you get started with Ghost
 
 
 
-25 Sep 2023
```

Comment:
@sam-lord There's a hardcoded date here - can you flag with Product to fix this?